sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -138,25 +129,25 @@ object DateTimeUtils {
    * Converts the timestamp to milliseconds since epoch. In spark timestamp values have microseconds
    * precision, so this conversion is lossy.
    */
-  def toMillis(us: SQLTimestamp): Long = {
+  def toMillis(us: Long): Long = {
     // When the timestamp is negative i.e before 1970, we need to adjust the millseconds portion.
     // Example - 1965-01-01 10:11:12.123456 is represented as (-157700927876544) in micro precision.
     // In millis precision the above needs to be represented as (-157700927877).
     Math.floorDiv(us, MICROS_PER_MILLIS)
   }
 
   /*
-   * Converts milliseconds since epoch to SQLTimestamp.
+   * Converts milliseconds since the epoch to microseconds.
    */
-  def fromMillis(millis: Long): SQLTimestamp = {
+  def fromMillis(millis: Long): Long = {
```

Comment:
ditto

Reply:
Renamed in https://github.com/apache/spark/pull/27618